sycl/unittests/event/GetWaitList.cpp

```diff
@@ -0,0 +1,54 @@
+//==--------------------- GetWaitList.cpp --- event unit test --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+#include <gtest/gtest.h>
+
+TEST(GetWaitList, GetWaitListTest) {
```

Comment:
Can we, for example, move the test to `sycl/unittests/queue` to avoid creating a new test suite? Or there is a reason for creating a new one?

Reply:
I thought it is more correct to create new directory for testing event functionality. But there is nothing blocking to move it to queue directory. I moved it to Wait.cpp